app/assets/javascripts/countries/views/show/CountryModalView.js

```diff
@@ -92,6 +103,8 @@ define([
 
     _cleanIndicators: function() {
       $('.indicators-group li').removeClass('is-selected');
+      $('#jurisdictions-list li').removeClass('is-selected');
```

Comment:
Ok, so I'd strongly recommend to cache the `#jurisdiction` object and then use ir looking into it instead of calling the list item and that stuff, do you want me to help? @andresgnlez